public/assets/ts/editor.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable-next-line */
```

Comment:
I tried passing Editor to EditorLoader as a module import to avoid this, but the editor on the page was broken and I wasn't bothered to try going down that hole to try and fix it using imports, but I agree we should try and avoid disabling eslint unless necessary